### PR TITLE
Add geocoder support for navidata.pl service 

### PR DESCRIPTION
**Pull request by navidata**
Recreating pull request due to technical problems. All discussed code parts from #96 have been fixed.

This pull request adds support for navidata.pl geocoding service. NaviData geocodes (and reverse geocodes) locations in Poland - Europe will be supported soon. API and services are free for non-commercial purposes.

### CODE REVIEW

**Comment by navidata** [comment]
hi!

Could you please verify if this code is ok for merge? 

best regards.

File: geopy/geocoders/navidata.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+            )
+        if status_code == 200:
+            # When there are no results, just return.
+            return
+
+        if status_code == 403:
+            raise GeocoderQueryError(
+                'Your request was denied.'
+            )
+        else:
```
Comment (ijl) [inline geopy/geocoders/navidata.py]: These shoudl be `elif`s. WPerhaps check `200` first?

File: geopy/geocoders/opencage.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 """
 
 from geopy.compat import urlencode
+from geopy.point import Point
```
Comment (ijl) [inline geopy/geocoders/opencage.py]: Unnecessary change?

File: geopy/geocoders/navidata.py [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+        """
+        Given a point, find an address.
+
+        :param query: The coordinates for which you wish to obtain the
+            closest human-readable addresses.
+        :type query: :class:`geopy.point.Point`, list or tuple of (latitude,
+            longitude), or string as "%(latitude)s, %(longitude)s"
+
+        :param boolean exactly_one: Return one result or a list of results, if
```
Comment (ijl) [inline geopy/geocoders/navidata.py]: Is `exactly_one` only invalid for `reverse`? In any case, I think the default value should be `True` then.

File: geopy/geocoders/navidata.py [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+
+        :param int timeout: Time, in seconds, to wait for the geocoding service
+            to respond before raising a :class:`geopy.exc.GeocoderTimedOut`
+            exception. Set this only if you wish to override, on this call
+            only, the value set during the geocoder's initialization.
+
+        """
+        params = {
+            'lat': self._coerce_point_to_string(query).split(',')[0],
+            'lon': self._coerce_point_to_string(query).split(',')[1]
```
Comment (ijl) [inline geopy/geocoders/navidata.py]: We only need to call `_coerce_point_to_string` once.

File: geopy/geocoders/navidata.py [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+            :class:`urllib2.ProxyHandler`.
+
+        """
+        super(NaviData, self).__init__(
+            scheme="http", timeout=timeout, proxies=proxies
+        )
+
+        self.api_key = api_key
+        self.domain = domain.strip('/')
+        #self.scheme = scheme
```
Comment (ijl) [inline geopy/geocoders/navidata.py]: Dead code.

**Comment by ijl** [comment]
Some comments inline, and merge conflicts (please fix by rebasing, for commit history). Also, I'd really like the test coverage on this to be 100%. Check with `nosetests -v --with-cover --cover-package=geopy test.geocoders.navidata`.

File: geopy/geocoders/navidata.py (hunk 5 above)
Comment (navidata) [inline geopy/geocoders/navidata.py]: what do you mean by dead code?
